src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -497,7 +497,18 @@ namespace winrt::TerminalApp::implementation
     // - <none>
     void TerminalPage::_OnExportChatHistoryRequested(const IInspectable& /*sender*/, const winrt::hstring& text)
     {
-        _SaveFileHelper(text, L"", L"");
+        auto now = std::chrono::system_clock::now();
+        auto time = std::chrono::system_clock::to_time_t(now);
+
+        std::tm local_time;
+        localtime_s(&local_time, &time);
+
+        std::stringstream ss;
+        ss << std::put_time(&local_time, "%m%d%Y-%H%M");
```

Comment:
There are only a hand full of countries in the world which use the MM DD YYYY date format. Something like this would respect the users settings:
```cpp
SYSTEMTIME time;
GetLocalTime(&time);

wchar_t dateBuf[256];
wchar_t timeBuf[256];
const auto dateLen = GetDateFormatEx(nullptr, 0, &time, nullptr, &dateBuf[0], ARRAYSIZE(dateBuf), nullptr);
const auto timeLen = GetTimeFormatEx(nullptr, 0, &time, nullptr, &timeBuf[0], ARRAYSIZE(timeBuf));

const std::wstring_view date{ &dateBuf[0], gsl::narrow_cast<size_t>(dateLen) };
const std::wstring_view time{ &timeBuf[0], gsl::narrow_cast<size_t>(timeLen) };

fmt::format(FMT_COMPILE(L"{} {}"), date, time);
```

However, I think RFC 3339 would be just as good if not better. Something like this:
```cpp
time_t nowTime;
time(&nowTime);

tm nowTm;
gmtime_s(&nowTm, &nowTime);

wchar_t buf[256];
size_t len = wcsftime(&buf[0], ARRAYSIZE(buf), L"%FT%T%z", &nowTm);
```

Reply:
Ah it's YY MM DD now. The above time.h still has the advantage that it's not another re-introduction of stringstream.

Reply:
This doesn't get the local time though right?

Reply:
nvm replacing `gmtime_s` with `localtime_s` fixes that, updated! thanks